core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2372,10 +2372,18 @@ private[spark] object Utils extends Logging {
 
     val serviceString = if (serviceName.isEmpty) "" else s" '$serviceName'"
     val maxRetries = portMaxRetries(conf)
+    val customPortEnable = conf.get(config.CUSTOM_SERVICE_PORT)
```

Comment:
So I guess you wan't this configuration anymore since we now can check if `originPort` is set or not.  e.g.) `originPort.isDefined`

Reply:
Thank you very much for your careful review, this configuration is indeed redundant.